modules/clientmanagements/client/controllers/list-clientmanagements.client.controller.js

```diff
@@ -11,19 +11,79 @@
     var vm = this;
 
     vm.clientmanagements = ClientmanagementsService.query();
-    vm.headSort = 'name';
+    vm.headSort= 'name';
```

Comment:
The arrows for each column should only be visible when sorting by that column. E.g. when you click name, the arrow appears, then when you click another column, the name arrow dissapears and the new column's appears.